crates/bevy_render/src/camera/camera.rs

```diff
@@ -349,20 +363,36 @@ impl RenderTarget {
             RenderTarget::Image(image_handle) => {
                 images.get(image_handle).map(|image| image.texture_format)
             }
+            RenderTarget::InitialWindow => {
+                let window_id = windows
+                    .initial
+                    .expect("expected an initial window to exist if going to retrieve format");
```

Comment:
Ditto here: this error message doesn't really explain how you might fix this / what went wrong.